templates/patterns/content_banner/pattern-content-banner.html.twig

```diff
@@ -23,7 +23,7 @@
 {% set _badges = [] %}
 {% for _badge in badges %}
   {% set _badge = _badge|merge({
-    'outline': true,
+    'outline': _badge.outline|default(true),
```

Comment:
This doesn't work, it always shows outlined entries:
![image](https://user-images.githubusercontent.com/8488617/223195251-df82185c-1ffa-4c93-970b-19804f1a2168.png)
The correct check is `'outline': _badge.outline ?? true,` to preserve false values.

Reply:
yes, thanks. changed them